### DISCUSSION TITLE
Picker Item with dangerouslySetInnerHTML

### DISCUSSION THREAD

**Discussion by ZoltanSzilagyiCse** [adobe/react-spectrum#1662] [Q&A] [answered]
Is it possible to use `Picker` elements (`Item`s) with `dangerouslySetInnerHTML`? For me it gives the following result:

![image](https://user-images.githubusercontent.com/14167557/110024951-0ffb6280-7d2f-11eb-9ce1-33c2d7825525.png)

Code:
```js
<Picker aria-label="X axis">
    {getInputFeatures()
        .flatMap(input => input.inputs)
        .map(input => (<Item key={input.id} textValue={input.name}>
            <div dangerouslySetInnerHTML={{__html: input.name}}/>
        </Item>))})}
</Picker>
```

The data:

```json
{
   "inputs":[
      {
         "id":"Lambda1",
         "name":"λ<sub>1</sub>"
      },
      {
         "id":"Lambda2",
         "name":"λ<sub>2</sub>"
      },
      {
         "id":"Mu",
         "name":"µ"
      },
      {
         "id":"c",
         "name":"c"
      },
      {
         "id":"t",
         "name":"t"
      },
      {
         "id":"r",
         "name":"r"
      },
      {
         "id":"n",
         "name":"n"
      }
   ]
}
```

**Comment by LFDanLu** [comment, marked as answer]
@ZoltanSzilagyiCse Can you try wrapping the `<div dangerouslySetInnerHTML={{__html: input.name}}/>` within a `<Text>` (https://react-spectrum.adobe.com/react-spectrum/Text.html)? This allows the proper `grid-area` to be set on the element wrapping the text, positioning the element into the appropriate grid section

**Reply by ZoltanSzilagyiCse** [reply]
Woah, I never thought about this solution. Thanks, with this one, it's working! 🙇‍♂️
